parser_test.go

```diff
@@ -0,0 +1,61 @@
+package manta
+
+import (
+	"bytes"
+	"compress/bzip2"
+	"io"
+	"net"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStreamingReaderTimeout(t *testing.T) {
+	assert := assert.New(t)
+
+	// create a http client with a timeout too low to get the entire replay
+	client := http.Client{
+		Timeout: 2 * time.Second,
+	}
+
+	// request the replay, the timeout should be enough for it to begin successfully
+	resp, err := client.Get("http://replay111.valve.net/570/2526759078_1821332888.dem.bz2")
```

Comment:
A small fraction have expired, but you can still get most of them.  Also, sometimes China replays download really slowly.